Cargo.toml

```diff
@@ -9,24 +9,28 @@ edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
-[lib]
-crate-type = ["cdylib", "rlib"]
-
 [dependencies]
 rand = "0.8"
 device_query = "1.1"
-eframe = "0.19" # Gives us egui, epi and web+native backends
+eframe = "0.19"                                                     # Gives us egui, epi and web+native backends
```

Comment:
Wonky whitespace

Reply:
ya I don't know, that's probably some toml formatter I have.